cookbooks/aws-parallelcluster-install/resources/package_repos/package_repos.rb

```diff
@@ -29,11 +29,7 @@
     # the epel recipe doesn't work on aarch64, needs epel-release package
     package 'epel-release' if node['platform_version'].to_i == 7 && node['kernel']['machine'] == 'aarch64'
 
-    unless node['platform_version'].to_i < 7
-      execute 'yum-config-manager_skip_if_unavail' do
-        command "yum-config-manager --setopt=\*.skip_if_unavailable=1 --save"
-      end
-    end
+    action_centos_skip_if_unavail
```

Comment:
Do we still need it here?

Reply:
I think that the goal behind this is to not block the installation of packages if some repository is temporarily offline.

Reply:
Why only on CentOS7 though?